drivers/sensor/isl29035/isl29035_trigger.c

```diff
@@ -84,8 +84,10 @@ static void isl29035_thread_cb(struct device *dev)
 	u8_t val;
 
 	/* clear interrupt */
-	i2c_reg_read_byte(drv_data->i2c, ISL29035_I2C_ADDRESS,
-			  ISL29035_COMMAND_I_REG, &val);
+	if (i2c_reg_read_byte(drv_data->i2c, ISL29035_I2C_ADDRESS,
+			      ISL29035_COMMAND_I_REG, &val) < 0) {
+		return;
```

Comment:
It will be good if you add a debug log to indicate the error. Something like below:

`LOG_DBG("isl29035: error reading command register");`

Reply:
Do you think SYS_LOG_ERR will be more appropriate?

Reply:
SYS_LOG_ERR is now deprecated. Instead use a new logging subsystem. Checkout https://docs.zephyrproject.org/latest/subsystems/logging/logger.html.